tests/db/test_bigchain_api.py

```diff
@@ -371,8 +371,8 @@ def test_invalid_transactions_in_block(self, b, user_public_key, ):
         }
 
         block_data = util.serialize(block)
-        block_hash = hash_data(block_data)
-        block_signature = PrivateKey(b.me_private).sign(block_data)
+        block_hash = crypto.hash_data(block_data)
+        block_signature = crypto.SigningKey(b.me_private).sign(block_data)
```

Comment:
If `SigningKey` is used to sign data, why it's not just a function? Why do we need an object here?